src/module/authentication/container/login-container.tsx

```diff
@@ -23,13 +24,24 @@ export default class LoginContainer extends React.Component<any> {
 			});
 		};
 
+		const resendEmail = () => {
+			AuthStore.resendActivationEmail(email)
+				.then((success) => {
+					if(!success) {
+						alert('Too many attempts to resend the activation email in a short period of time, please try again in 5 minutes');
+					}
+				})
+		};
```

Comment:
This same resendEmail function is also defined in the AccountCreatedContainer.  Maybe move it to a utility file so it can be shared?